### PR TITLE
SDR: the adventure continues

### PR DESCRIPTION
**Pull request by porcuquine**
- [x] Specify `[]CommP` -> `CommD` verification via `ComputeRootPieceInfo`.
- [x] Specify vanilla verification.
- [x] Specify supersectors.

### CODE REVIEW

File: src/systems/filecoin_mining/sector/sealing.id [hunk 1]
```diff
@@ -22,20 +22,23 @@ type SealCfg struct {
 // needs to verify a Seal.
 type SealVerifyInfo struct {
     SectorID
-    OnChain     OnChainSealVerifyInfo
+    OnChain                OnChainSealVerifyInfo
     SealCfg
-    PieceInfos  [PieceInfo]
+    Randomness             SealRandomness
+    InteractiveRandomness  SealRandomness
```
Comment (nicola) [inline src/systems/filecoin_mining/sector/sealing.id]: they should be two different types

File: src/systems/filecoin_mining/sector/sealing.id [hunk 2]
```diff
@@ -22,20 +22,23 @@ type SealCfg struct {
 // needs to verify a Seal.
 type SealVerifyInfo struct {
     SectorID
-    OnChain     OnChainSealVerifyInfo
+    OnChain                OnChainSealVerifyInfo
     SealCfg
-    PieceInfos  [PieceInfo]
+    Randomness             SealRandomness
```
Comment (nicola) [inline src/systems/filecoin_mining/sector/sealing.id]: this should be SealSeed?

File: src/systems/filecoin_mining/sector/sealing.id (hunk 1 above)
Comment (porcuquine) [reply src/systems/filecoin_mining/sector/sealing.id]: Done.

File: src/systems/filecoin_mining/sector/sealing.id (hunk 2 above)
Comment (porcuquine) [reply src/systems/filecoin_mining/sector/sealing.id]: As discussed elsewhere, `SealSeed` replaces `ReplicaId`.

File: src/systems/filecoin_mining/storage_proving/sealer/sealer.id [hunk 3]
```diff
@@ -4,20 +4,20 @@ import addr "github.com/filecoin-project/specs/systems/filecoin_vm/actor/address
...
-    RandomSeed    sector.SealRandomness
-    UnsealedPath  file.Path
-    SealedPath    file.Path
-    DealIDs       [deal.DealID]
+    SectorID       sector.SectorID
+    SealCfg        sector.SealCfg
+    MinerID        addr.Address
+    RandomSeed     sector.SealRandomness
+    UnsealedPaths  [file.Path]
+    SealedPaths    [file.Path]
```
Comment (jbenet) [inline src/systems/filecoin_mining/storage_proving/sealer/sealer.id]: why are these multiple paths now? (for one sector)

Comment (jbenet) [reply src/systems/filecoin_mining/storage_proving/sealer/sealer.id]: may be good to add a comment about it

File: src/systems/filecoin_mining/sector/sealing.id [hunk 4]
```diff
@@ -22,20 +22,23 @@ type SealCfg struct {
...
 // state tree but will be verified in sm.CommitSector. See SealCommitment for
 // data stored on the state tree for each sector.
 type OnChainSealVerifyInfo struct {
-    SealedCID     SealedSectorCID  // CommR
-    Epoch         block.ChainEpoch
-    Proof         SealProof
-    DealIDs       [deal.DealID]
+    SealedCID         SealedSectorCID  // CommR
+    Epoch             block.ChainEpoch
+    InteractiveEpoch  block.ChainEpoch
```
Comment (jbenet) [inline src/systems/filecoin_mining/sector/sealing.id]: - is this the `ChallengeEpoch`?  (`InteractiveEpoch` not super clear)
- can we name both epochs well to distinguish them?

Comment (jbenet) [reply src/systems/filecoin_mining/sector/sealing.id]: and maybe add one-line comment

**Comment by jbenet** [comment]
Rebased (for merging)

**Pull request merged**